dashboard/src/components/DeploymentStatus/DeploymentStatus.tsx

```diff
@@ -40,8 +48,11 @@ class DeploymentStatus extends React.Component<IDeploymentStatusProps> {
     const { deployments } = this.props;
     if (deployments.length > 0) {
       return deployments.every(d => {
-        const status: IDeploymentStatus = d.status;
-        return status.availableReplicas === status.replicas;
+        if (d.item) {
+          const status: IDeploymentStatus = d.item.status;
```

Comment:
This is the first time I see the every method. Interesting. You might want to explain that `every` returns false at the moment that the callback returns falsey.

Reply:
isn't it quite self-explanatory?

Reply:
Well, it was not for me, but that can be my fault :)